### PR TITLE
Add suggestion for `arg` & `args`

### PR DESCRIPTION
**Pull request by July541**
As mentioned in #1247, it would be great if we can warn people if they are passing `@@` to `arg` or `args`.

### CODE REVIEW

File: libafl/src/executors/command.rs [hunk 1]
```diff
@@ -479,12 +479,16 @@ impl CommandExecutorBuilder {
     }
 
     /// Adds an argument to the program's commandline.
+    ///
+    /// You may want to use [`CommandExecutor::parse_afl_cmdline`] if you're going to pass `@@`
```
Comment (domenukk) [inline libafl/src/executors/command.rs:483]: Do we want to explain here what `@@` means / the AFL context?

File: libafl/src/executors/command.rs [hunk 2]
```diff
@@ -489,6 +490,7 @@ impl CommandExecutorBuilder {
     /// Adds a range of arguments to the program's commandline.
     ///
     /// You may want to use [`CommandExecutor::parse_afl_cmdline`] if you're going to pass `@@`
+    /// represents the input file generated by the fuzzer.
```
Comment (domenukk) [inline libafl/src/executors/command.rs:493]: The line breaks are not in the compiled doc, so it'll look like this: 
`if you're going to pass `@@` represents the input file generated by the fuzzer.`
I don't want to nag too much, but maybe we should write something more clear, like
```
if you're going to pass `@@` to represent the input file generated by the fuzzer (similar to the `afl-fuzz` command line).
```

Comment (July541) [reply libafl/src/executors/command.rs]: > The line breaks are not in the compiled doc

Yes, that's what I want.

> but maybe we should write something more clear

Done, please let me know if any improvements, it's really important for newcomers!

Comment (July541) [reply libafl/src/executors/command.rs]: Me neither😄, both are acceptable to me. Keep the current since this pr has been merged.

**Pull request merged**